assets/data/peopleParts/drauvenHead_haunt.json

```diff
@@ -150,5 +150,54 @@
        "ifNoOwnerAspects": ["themePiece_crow_head", "themePiece_wolf_head", "themePiece_tree_head", "themePiece_frog_head"],
        "facialExpression": ["hit", "dead"]
     }
+      {
```

Comment:
Layer definitions look good, but when we put them into the `peopleParts` files they'll appear on the heads in the previews regardless of whether the character has the qualifying aspect or not - that's why I put them in to the `drauvenSkin` file originally. :)

Reply:
fixed!